packages/fuel-indexer/src/service.rs

```diff
@@ -134,7 +134,7 @@ impl IndexerService {
         .await?;
 
         let mut items = vec![
-            (IndexerAssetType::Wasm, exec_source.to_vec()),
+            (IndexerAssetType::Wasm, Vec::<u8>::from(exec_source)),
```

Comment:
Just a suggestion: Perhaps we can give `items` a type and then use `Into` on each of the bits as necessary?

```
let mut items: Vec<(IndexerAssetType, Vec<u8>)> = vec![
    (IndexerAssetType::Wasm, exec_source.into())),
    ...
];
```

Reply:
Me like this idea